build-docs.sh

```diff
@@ -10,7 +10,7 @@ mkdir -p docs/docs dockerout
 # All protocol buffer files
 # These include a number of proto files that aren't part of the API, notably persisted_trace.proto
 # and ingest_job.proto. Those should probably be moved out of this repository.
-PROTOFILES=$(find $PWD/cognite/seismic/protos -name '*.proto' -printf "protos/cognite/seismic/protos/%P ")
+PROTOFILES=$(find $PWD/cognite/seismic/protos -name '*.proto' | sed "s~$PWD/cognite/seismic/protos~protos/cognite/seismic/protos~")
```

Comment:
This is just so that BSD systems can also run this script :)

Reply:
oh right osx has a bsd userspace...